### DISCUSSION TITLE
PDF to jpg

### DISCUSSION THREAD

**Discussion by AndiLeni** [libvips/libvips#3084] [Q&A] [answered]
Hello, 

I am trying to replace ImageMagick with libvips for converting pdf files to jpg.
Using `vips copy "files/test.pdf[n=-1]" a.jpg` works fine so far, but the resolution is not high enough.
Is it possible to increase the output resolution of the jpg?

Thanks

**Comment by jcupitt** [comment, marked as answer]
Hi @AndiLeni,

The PDF loader defaults to 72 DPI, but you can change it, eg.:

```
vips copy "files/test.pdf[n=-1,dpi=300]" a.jpg
```

You can also set a scale for relative adjustment, see:

https://www.libvips.org/API/current/VipsForeignSave.html#vips-pdfload

**Reply by AndiLeni** [reply]
This is exactly what I was looking for. Thank you very much :-)
